integration-tests/manifests/annotations/validate_annotations_test.go

```diff
@@ -0,0 +1,1003 @@
+	if err != nil {
+		t.Errorf("Failed to get deployment app: %s", err.Error())
+	}
+
+	// List pods belonging to the deployment
+	set := labels.Set(deployment.Spec.Selector.MatchLabels)
+	deploymentPods, err := clientSet.CoreV1().Pods(uniqueNamespace).List(context.TODO(), metav1.ListOptions{
+		LabelSelector: set.AsSelector().String(),
+	})
+	if err != nil {
+		t.Errorf("Error listing pods for deployment: %s", err.Error())
+	}
+
+	//wait for pods to update
+	if !checkIfAnnotationExists(deploymentPods, []string{injectJavaAnnotation, autoAnnotateJavaAnnotation, injectPythonAnnotation, autoAnnotatePythonAnnotation}) {
+		t.Error("Incorrect Annotations")
+	}
+
+}
+
+// ---------------------------USE CASE 2 (Java on Deployment and Python Should be Removed)------------------------------
+func TestUseCase2(t *testing.T) {
+
+	t.Parallel()
+	clientSet := setupTest(t)
+	uniqueNamespace := "sample-namespace-2"
+	if err := createNamespaceAndApplyResources(t, clientSet, uniqueNamespace, []string{"sample-deployment.yaml"}); err != nil {
+		t.Fatalf("Failed to create/apply resoures on namespace: %v", err)
+	}
+
+	defer func() {
+		if err := deleteNamespaceAndResources(clientSet, uniqueNamespace, []string{"sample-deployment.yaml"}); err != nil {
+			t.Fatalf("Failed to delete namespaces/resources: %v", err)
+		}
+	}()
+
+	annotationConfig := auto.AnnotationConfig{
+		Java: auto.AnnotationResources{
+			Namespaces:   []string{""},
+			DaemonSets:   []string{""},
+			Deployments:  []string{filepath.Join(uniqueNamespace, deploymentName)},
+			StatefulSets: []string{""},
+		},
+		Python: auto.AnnotationResources{
+			Namespaces:   []string{""},
+			DaemonSets:   []string{""},
+			Deployments:  []string{""},
+			StatefulSets: []string{""},
+		},
+	}
+	jsonStr, err := json.Marshal(annotationConfig)
+	if err != nil {
+		t.Errorf("Failed to marshal: %v\n", err)
+	}
+
+	deployment, err := clientSet.AppsV1().Deployments(amazonCloudwatchNamespace).Get(context.TODO(), amazonControllerManager, metav1.GetOptions{})
+	if err != nil {
+		t.Errorf("Error getting deployment: %v\n", err)
+	}
+
+	//finding where index of --auto-annotation-config= is (if it doesn't exist it will be appended)
+	updateAnnotationConfig(deployment, string(jsonStr))
+	if !updateOperator(t, clientSet, deployment) {
+		t.Error("Failed to update Operator")
+	}
+
+	//check if deployment has annotations.
+	deployment, err = clientSet.AppsV1().Deployments(uniqueNamespace).Get(context.TODO(), deploymentName, metav1.GetOptions{})
+	if err != nil {
+		if err != nil {
+			t.Errorf("Error listing pods for deployment: %s", err.Error())
+		}
+	}
+
+	// List pods belonging to the deployment
+	set := labels.Set(deployment.Spec.Selector.MatchLabels)
+	deploymentPods, err := clientSet.CoreV1().Pods(uniqueNamespace).List(context.TODO(), metav1.ListOptions{
+		LabelSelector: set.AsSelector().String(),
+	})
+	if err != nil {
+		t.Error("Failed to update Operator")
```

Comment:
Can we also append the error message here in case the API call is broken and we know what was the issue.